pkg/scheduler/framework/plugins/podtopologyspread/pod_topology_spread.go

```diff
@@ -118,11 +121,9 @@ func (pl *PodTopologySpread) RemovePod(ctx context.Context, cycleState *framewor
 func getPodTopologySpreadMetadata(cycleState *framework.CycleState) (*predicates.PodTopologySpreadMetadata, error) {
 	c, err := cycleState.Read(preFilterStateKey)
 	if err != nil {
-		return nil, err
-	}
-
-	// It's possible that meta is set to nil intentionally.
-	if c == nil {
+		// The metadata wasn't pre-computed in prefilter. We ignore the error for now since
+		// we are able to handle that by computing it again (e.g. in Filter()).
```

Comment:
is this actually true for pod topology spread?

Reply:
The nil meta will be caught in the Filter, so yes, it should be good.

Reply:
sure, but we are not computing it again now.